app/views/works/index.html.erb

```diff
@@ -0,0 +1,132 @@
+<h2>List of works</h2>
+<% if @vote != nil %>
+  <%= render partial: "layouts/error_messages", locals: { model: @vote } %>
+  <% end %>
+
+<h3>Music Albums</h3>
+<table>
+
+  <th>
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?